app/templates/components/events/view/export/download-common.hbs

```diff
@@ -0,0 +1,19 @@
+<h3 class="ui header">
+  <div class="content">
+    {{t 'Download Event as'}} {{downloadType}}
+  </div>
+  <div class="sub header">
+    {{t 'Once the event is live and the schedule is published,'}} {{downloadType}} {{t ' version of the schedule will be available at:'}}
+  </div>
+</h3>
+<div class="ui hidden divider"></div>
+<div class="ui secondary segment url">
+  <pre>http://localhost:5000/e/280a30d7/schedule/{{downloadUrl}}</pre>
+</div>
+<div class="content">
+  {{t 'Alternatively, you can download the'}} {{downloadType}} {{t ' here'}}
+</div>
+<div class="ui hidden divider"></div>
```

Comment:
Remove this and the button. Instead make `download the schedule.ical here` a link